.ci/jenkins_job_build.sh

```diff
@@ -7,8 +7,18 @@
 
 set -e
 
+export GOPATH=${WORKSPACE}/go
+# For all our jobs  will install golang from static binaries
```

Comment:
propose to rephrase: `# For all our jobs, golang will be installed using static binaries`